### PR TITLE
post: pro pack release

### PR DESCRIPTION
**Pull request by cexbrayat**

### CODE REVIEW

File: serve.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # /bin/sh
 
-docker run -it -v $PWD:/srv/jekyll -p 4000:4000 jekyll/pages jekyll serve --force_polling
+docker run -it -v $PWD:/srv/jekyll -p 4000:4000 jekyll/jekyll:pages bundle exec jekyll serve --force_polling --incremental --future
```
Comment (cexbrayat) [inline serve.sh]: this fixes the docker image, but it's slow as hell. I had to install ruby 😢

**Comment by cexbrayat** [comment]
Updated with Agnes' feedback

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Install Github pages :
 
 Run the blog :
 
-    jekyll serve --watch
+    jekyll serve --watch --incremental --future
```
Comment (cexbrayat) [inline README.md]: `--future` allows to display future posts (it seems that was the case by default previously but not anymore)
`--incremental` make the regenration blazing fast (0.2 second vs 4 seconds previously)

**Comment by clacote** [comment]
Ajout de la version FR: 48d86fd
Il restera à rajouter un petit lien entre les deux pages.

**Comment by clacote** [comment]
Link between language versions: 	8ebe912

File: _posts/2016-09-07-angular-2-pack-pro-francais.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: Notre Pro Pack Angular 2 est disponible !
+author: cexbrayat
+tags: ["Angular 2", "ebook"]
+description: "Notre Pro Pack est disponible et vous permettra de construire une application complète pour tout apprendre sur Angular 2."
+---
+
+*Looking for the English version? It's [there](/2016/09/07/angular-2-pro-pack-english/).*
+{: style="text-align: right;"}
```
Comment (clacote) [inline _posts/2016-09-07-angular-2-pack-pro-francais.md]: Yes, this is the markdown way to add HTML attributes to generated HTML. This is a maruku feature: http://maruku.rubyforge.org/proposal.html#attribute_lists

**Pull request merged**